src/draw/vg_lite/lv_draw_vg_lite_label.c

```diff
@@ -162,7 +162,6 @@ static void draw_letter_cb(lv_draw_task_t * t, lv_draw_glyph_dsc_t * glyph_draw_
 #if LV_USE_FREETYPE
             case LV_FONT_GLYPH_FORMAT_VECTOR: {
                     if(lv_freetype_is_outline_font(glyph_draw_dsc->g->resolved_font)) {
-                        glyph_draw_dsc->glyph_data = lv_font_get_glyph_bitmap(glyph_draw_dsc->g, glyph_draw_dsc->_draw_buf);
```

Comment:
This PR: https://github.com/lvgl/lvgl/pull/7560 has called `lv_font_get_glyph_bitmap` in advance. If the GPU backend calls it again, the cache reference/dereference times will be unpaired, resulting in the inability to kick out the cache.